client/extensions/woocommerce/index.js

```diff
@@ -14,31 +14,88 @@ import ProductCreate from './app/products/product-create';
 import Dashboard from './app/dashboard';
 import StatsController from './app/stats/controller';
 
-const Controller = {
-	dashboard: function( context ) {
+function addStorePage( storePage ) {
+	page( storePage.route, siteSelection, navigation, function( context ) {
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/13634/commits/ceb4f5e247c7646489a00d8ba96def88f368f0d6